src/main/target/OMNIBUSF4NANOV7/target.h

```diff
@@ -39,17 +39,33 @@
 #define SPI3_MISO_PIN           PB4
 #define SPI3_MOSI_PIN           PB5
 
+#define USE_MULTI_GYRO
+
 #define USE_GYRO
 #define USE_GYRO_SPI_MPU6500
-
-#define GYRO_1_CS_PIN           PD2
-#define GYRO_1_SPI_INSTANCE     SPI3
+#define USE_GYRO_SPI_MPU6000
 
 #define USE_ACC
 #define USE_ACC_SPI_MPU6500
+#define USE_ACC_SPI_MPU6000
+
+#define GYRO_1_CS_PIN           PD2
+#define GYRO_1_SPI_INSTANCE     SPI3
 
 #define GYRO_1_ALIGN            CW0_DEG
 
+#define GYRO_2_CS_PIN           PC4
+#define GYRO_2_SPI_INSTANCE     SPI1
+
+#define GYRO_2_ALIGN            CW0_DEG
+
+// MPU6000 interrupts
+#define USE_EXTI
```

Comment:
Why are you adding this? Seeing that there do not seem to be any connected EXTI pins, compiling in EXTI support seems to be pointless.